spec/support/helpers/take_at_most_helper.rb

```diff
@@ -0,0 +1,21 @@
+module TakeAtMostHelper
+  # Assert that it takes at most a certain amount of time to run a block.
+  #
+  # @example
+  #  # Assert that it takes at most 1 second to run the block
+  #  take_at_most(1) { sleep 0.5 }
+  #
+  # @param time [Integer] The maximum amount of time the block is allowed to
```

Comment:
I see it being used with a float too.
```suggestion
  # @param time [Integer, Float] The maximum amount of time the block is allowed to
```